snyk-monitor/values.yaml

```diff
@@ -28,3 +28,12 @@ clusterName: ""
 # The snyk-monitor requires disk storage to temporarily pull container images and to scan them for vulnerabilities.
 # This value controls how much disk storage _at most_ may be allocated for the snyk-monitor. The snyk-monitor mounts an emptyDir for storage.
 temporaryStorageSize: 50Gi
+
+# CPU/Mem requests and limits for snyk-monitor
+requests:
```

Comment:
can we save the exact same format? so everything is a string like before. so change `250m` to `'250m'`, `400Mi` to `'400Mi'`, `"1"` to `'1'` and `2Gi` to `'2Gi'`

Reply:
@shaimendel Sure, definitely happy to. This would keep things inconsistent with other items in the `values.yaml` file however, would you like me to change all of those as well?

Reply:
not for now, let's just change the ones you moved as a beginning. thank you!